src/components/Characters/Characters.js

```diff
@@ -1,9 +1,34 @@
+import { useState } from "react"
 import CharacterCard from "../CharacterCard/CharacterCard"
 import "./Characters.css"
 
-export default function Characters({characters, selectCharacter, player1, player2, displayFight}){
+export default function Characters({setDropDownInput, dropDownInput, filterCharacters, characters, selectCharacter, player1, player2, displayFight, getCharacters}){
+  const [userInput, setUserInput] = useState('')
   let displayedCharacters = characters.map(character=>{
     character.key = character.id
+    if(dropDownInput === 'other' && character.publisher !== 'marvel' && character.publisher !== 'DC Comics'){
+    return (
+      <CharacterCard 
+        name={character.name}
+        image={character.image}
+        id={character.id}
+        key={character.id}
+        character={character}
+        selectCharacter={selectCharacter}
+      />
+    )} 
+  if(character.publisher===dropDownInput){
+    return (
+      <CharacterCard 
+        name={character.name}
+        image={character.image}
+        id={character.id}
+        key={character.id}
+        character={character}
+        selectCharacter={selectCharacter}
+      /> 
+    )} 
+  if (dropDownInput === 'all') {
```

Comment:
We could probably use a switch statement here instead of a long conditional, but if it works, it works! We can also keep this in mind when refactoring!